vim_menutrans_helper.py

```diff
@@ -0,0 +1,228 @@
+#!/user/bin/env python3
+
+import re
+import glob
+import os
+import sys
+
+menu_commands = {"me", "menu", "noreme", "noremenu", "am",
+             "amenu", "an", "anoremenu", "nme", "nmenu",
+             "nnoreme", "nnoremenu", "ome", "omenu", "onoreme",
+             "onoremenu", "vme", "vmenu", "vnoreme", "vnoremenu",
+             "xme", "xmenu", "xnoreme", "xnoremenu", "sme", "smenu",
+             "snoreme", "snoremenu", "ime", "imenu", "inoreme", "inoremenu",
+             "cme", "cmenu", "cnoreme", "cnoremenu", "tlm", "tlmenu", "tln",
+             "tlnoremenu", "tunmenu", "tu", "popup", "popu"}
+toolbar_commands = {"tmenu", "tm"}
+exe_commands = {"exe", "exec", "execute"}
+sil_commands = {"sil", "silent", "sil!", "silent!"}
+
+def unescape_double_quotes(origin):
+    """Unescape a string inside double quotes."""
+    return (origin.replace('\\"', '"').replace("\\'", "'").replace('\\t', '\t').
+            replace('\\\\', '\\'))
+
+def unescape_single_quotes(origin):
+    """Unescape a string inside single quotes."""
+    return origin.replace("''", "'")
+
+def make_untranslated_dict(untranslation_file, untranslated_dict):
+    """Get the diffence between untranslated file and translated file."""
```

Comment:
This method doesn't get the differences. It only extract the messages to be translated. Please update the command. Also, rename the method name to `extract_messages()` to make this clear.